OneSignalSDK/onesignal/src/main/java/com/onesignal/OneSignalCacheCleaner.java

```diff
@@ -4,69 +4,218 @@
 import com.onesignal.OneSignalDbContract.CachedUniqueOutcomeNotificationTable;
 
 import android.content.Context;
+import android.database.Cursor;
 import android.database.sqlite.SQLiteDatabase;
 import android.os.Process;
+import android.support.annotation.WorkerThread;
+
+import org.json.JSONException;
+
+import java.util.Set;
 
 class OneSignalCacheCleaner {
 
-    private static String OS_DELETE_OLD_CACHED_DATA = "OS_DELETE_OLD_CACHED_DATA";
+    private final static long ONE_WEEK_IN_SECONDS = 604_800L;
+    private final static long SIX_MONTHS_IN_SECONDS = 15_552_000L;
```

Comment:
`SIX_MONTHS_IN_SECONDS` should be something more descriptive to what is it used for like `IAM_MAX_CACHE_SEC`

Reply:
6 months should be a comment instead.

Reply:
You are correct, on it